### DISCUSSION TITLE
Feature Request - Export Toolpaths as OBJ - PrusaSlicer

### DISCUSSION THREAD

**Discussion by mr-pewpy** [OrcaSlicer/OrcaSlicer#4532] [Ideas]
https://help.prusa3d.com/article/export-toolpaths-as-obj_1777

Can we add this to Orca ?

idk how to start but is it easy ?

**Comment by Blendist3D** [comment]
Was about to request this myself, I need these obj files for the 3D renders I make of our printed products, and I'm sure many other people would love this feature

**Comment by ghost** [comment]
I second this.

**Comment by ilko-k** [comment]
One more vote. I was about to submit the same request but a quick search made me land here.

An example of what we can do :
![Slice](https://github.com/user-attachments/assets/e6544499-4cf5-4499-923d-986eb49143d8)

**Comment by Cothek** [comment]
I fourth the request.

**Comment by Marcos-Reckers** [comment]
UP

**Comment by ezraclintoc** [comment]
Yeah we NEED this

I litterly i have to download prusa slicer JUST FOR THIS

**Comment by klw7890** [comment]
Yup! It's the only thing I use PrusaSlicer for at this point

**Comment by rindo-eiji** [comment]
Need this!

**Comment by Artemis1-0** [comment]
We need this pls.

**Comment by kleinwareio** [comment]
This exists now as of 2.3.2 and I can confirm that the `.obj` imports successfully into Blender 5.1.1

https://github.com/OrcaSlicer/OrcaSlicer/wiki/import_export#export-sliced

> Export Sliced
> Toolpaths as OBJ: Export the sliced model 3D preview as an OBJ file.

**Reply by Artemis1-0** [reply]
Unfortunately, this is right after I stopped needing it. Why is everything that i needed with 3d printing getting released or fixed as soon as I'm done? 😭😭😭
Great nonetheless.
